app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  def update
+    @user = User.find(params[:id])
+
+    @user.first_name = params[:user][:first_name]
+    @user.last_name = params[:user][:last_name]
+    @user.phone = params[:user][:phone]
```

Comment:
no :) go back and read more on the rails docs about this, and try again and we will discuss this after you do that...u won't list this out each time, can u imagine if you had 30 attributes on a class?